update-last-updated.js

```diff
@@ -0,0 +1,28 @@
+const {readFileSync, writeFileSync} = require('fs');
+
+const currentDate = new Date();
+const UTC = `Date.UTC(${currentDate.getUTCFullYear()}, ${currentDate.getUTCMonth()}, ${currentDate.getUTCDate()}, ${currentDate.getUTCHours()}, ${currentDate.getUTCMinutes()}, ${currentDate.getUTCSeconds()})`;
+console.log('\x1b[33m%s\x1b[0m', `Updating last updated config date to ${UTC}`);
+
+
+try {
+    console.log('\x1b[33m%s\x1b[0m', 'Reading file & updating ##LAST_UPDATED##')
+    let config = readFileSync(`${__dirname}/src/Config.ts`).toString();
```

Comment:
```suggestion
    let config = readFileSync(`${__dirname}/src/Config.ts`, 'utf-8');
```
so we don't have to convert buffer -> string